templates/base.html.twig

```diff
@@ -0,0 +1,167 @@
+                                    <li role="presentation"{% if app.request.attributes.get('_route') starts with 'ra_locations' %} class="active"{% endif %}>
+                                        <a href="{{ path('ra_locations_manage') }}">{{ 'ra.menu.ra_locations'|trans }}</a>
+                                    </li>
+                                {% endif %}
+
+                                {% if is_granted('ROLE_RAA') or is_granted('ROLE_SRAA') %}
+                                    <li role="presentation"{% if app.request.attributes.get('_route') starts with 'institution-configuration' %} class="active"{% endif %}>
+                                        <a href="{{ path('institution-configuration') }}">{{ 'ra.menu.institution-configuration'|trans }}</a>
+                                    </li>
+                                {% endif %}
+
+                                <li role="presentation"{% if app.request.attributes.get('_route') == 'ra_profile' %} class="active"{% endif %}>
+                                    <a href="{{ path('ra_profile') }}">{{ 'ra.menu.ra_profile'|trans }}</a>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="col-sm-4">
+                            <form name="logout" method="post" action="{{ logout_url('saml_based') }}" class="pull-right">
+                                <button type="submit" class="btn btn-link"><i class="fa fa-sign-out"></i> {{ 'button.logout'|trans }}</button>
+                            </form>
+                            {% set locale_switcher = stepup_locale_switcher(app.request.locale, 'ra_switch_locale', {'return-url': app.request.uri}) %}
+                            {{ form_start(locale_switcher, { attr: { class: 'form-inline' }}) }}
+                            {{ form_widget(locale_switcher.locale) }}
+                            {{ form_widget(locale_switcher.switch) }}
+                            {{ form_end(locale_switcher) }}
+                        </div>
+                    </div>
+                    <div class="clearfix"></div>
+                    <hr>
+                {% endif %}
+            {% endblock menu %}
+        {% endblock page_header %}
+
+        {% block form_errors %}
+            {% if app.session.flashbag.peekAll|length > 0 %}
+                <div class="row-fluid">
+                    <div id="flash" class="span12">
+                        {% for type, flashes in app.session.flashbag.all %}
+                            {% for flash in flashes %}
+                                <div class="alert alert-{{ type }} fade in">
+                                    {{ flash|trans }}
+                                </div>
+                            {% endfor %}
+                        {% endfor %}
+                    </div>
+                </div>
+            {% endif %}
+        {% endblock form_errors %}
```

Comment:
I see this solution differs from the one I chose when dropping MOPA support from SelfService. I think we should go for one solution (and maybe bundle it in a shared library ;) ) Joking aside. I think for maintainability we should stick with one solution and run with that. Let's discuss what solution is best.

Reply:
I've tried your solution, but that didn't result in a readable flash message:
https://github.com/OpenConext/Stepup-Gateway/pull/194/commits/ac41aea9a9a4033e01a156ab2612a856c20d8f1a#diff-a2f33b79364623e3cb2737239c5eee62R77